samples/Mvc.Client/Startup.cs

```diff
@@ -49,6 +51,24 @@ public class Startup {
                 options.ClientSecret = "98f1bf028608901e9df91d64ee61536fe562064b";
             });
 
+            app.UseLinkedInAuthentication(options => {
```

Comment:
Please revert these changes (removed in a previous commit). Only keep the Reddit provider so I can give it a try :+1: